server/channels/store/sqlstore/store.go

```diff
@@ -200,7 +200,7 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) (*Sql
 	store.stores.compliance = newSqlComplianceStore(store)
 	store.stores.session = newSqlSessionStore(store)
 	store.stores.oauth = newSqlOAuthStore(store)
-	store.stores.oauthOutgoingConnection = NewSqlOAuthOutgoingConnectionStore(store)
+	store.stores.OutgoingOAuthConnection = NewSqlOutgoingOAuthConnectionStore(store)
```

Comment:
The other properties here all start with lowercase. Maybe this should follow suit?

```suggestion
	store.stores.outgoingOAuthConnection = newSqlOutgoingOAuthConnectionStore(store)
```

Reply:
Good catch 👌